### PR TITLE
Users/grkuehnh/create anp with jupyter

### PR DESCRIPTION
**Pull request by GrKuehnH**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/systemlink-operations-handbook/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?
Added a chapter for Preview Features and added content for a TDM-Preview Feature "Creating and Uploading Analysis Automation Procedures with Jupyter Notebooks"
This will help customers try out the new preview feature.
The content of this topic will be removed as soon as the Preview status is lifted (2021 R2) and the topic publishes in Zoomin

NB: Posting posting on GitHub as a tech writer does not adhere to the guidelines of authorized people to add content. This is a topic I created but did not publish for 2021 R1 because the feature is still a preview feature. As a result of my research into the SystemLink operations handbook for a presentation, I wanted to try out the publishing process myself. This does not mean that tech writers will be posting content for the operations handbook in the future.

### Why should this Pull Request be merged?
The guidelines specifically mentions that content for preview features is desired in the operations handbook.

### What testing has been done?
-Passed the documentation review process. Also reviewed by devs and PO.
-Lintern Ok

### CODE REVIEW

File: handbook/preview-features/preview-features.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+This section contains guided help for preview features in the current SystemLink release. While preview features are not yet complete and have limited functionality, they provide an understanding of the direction the product is taking and give users the opportunity to experiment and give feedback.
+
+## Creating and Uploading Analysis Automation Procedures with Jupyter Notebooks
+
+Analysis Automation performs recurring analyses with similar data and exports the results as a PDF or graphics file for management reporting. Analysis automation procedures define the analyses and contain the analysis script, the parameter definition, and the search query for the respective data. Currently, you create procedures in DIAdem VBS or Python, but now you can also create them in Jupyter and upload them straight to Analyis Automation.  
+
+### Prerequisites
+
+To create analysis automation procedures with Jupyter in SystemLink, you need to install SystemLink 2021 R1 and the Measurement Data Analysis module.
```
Comment (LukasKall) [inline handbook/preview-features/preview-features.md:11]: In order to see the Jupyter UI, you also have to install the JupyterHub Module.

File: handbook/preview-features/preview-features.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    4. Select the __Execution Mode__.
+      Select __Compare__ if you want to run the analysis notebook on all data elements the search query retrieves.
+      Select __Parallel__ if you want to run the analysis notebook on each data element the search query retrieves.
+4. Click __OK__.
+    The procedure uploads to Analysis Automation.
+5. Click the __Analysis Automation__ link to view the procedure on the __Procedures__ tab in Analysis Automation.
+6. Approve the procedure for it to go live.  
+*Note: Approving procedures requires a separate privilege.*
+
+After uploading the procedure, [add a task to execute your procedure in Analysis Automation](https://www.ni.com/documentation/en/systemlink/2021r1/analysis/adding-editing-tasks-for-analysis-automation-procedures/).
```
Comment (LukasKall) [inline handbook/preview-features/preview-features.md:35]: Shouldn't the link be a redirect link so it doesn't include a version number?

File: handbook/preview-features/preview-features.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Analysis Automation performs recurring analyses with similar data and exports the results as a PDF or graphics file for management reporting. Analysis automation procedures define the analyses and contain the analysis script, the parameter definition, and the search query for the respective data. Currently, you create procedures in DIAdem VBS or Python, but now you can also create them in Jupyter and upload them straight to Analyis Automation.  
+
+### Prerequisites
+
+To create analysis automation procedures with Jupyter in SystemLink, you need to install SystemLink 2021 R1 and the Measurement Data Analysis module.
+
+### Steps to Take
+
+Create an analysis automation procedure from the analysis notebook you write in Jupyter and upload it to the procedures library in Analysis Automation.
+
```
Comment (JD-Robertson) [inline handbook/preview-features/preview-features.md:16]: Are there screenshots that would be helpful for this section?

**Review by JD-Robertson — APPROVED** [review]

File: handbook/preview-features/preview-features.md (hunk 2 above)
Comment (GrKuehnH) [reply handbook/preview-features/preview-features.md]: Fixed.

File: handbook/preview-features/preview-features.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+1. In __Jupyter__, open the analysis notebook you want to upload.
+
+2. Right-click the notebook and select __New Analysis Automation Procedure__.
+
+3. In the dialog box, specify the following information:
+    1. Enter a name and description for the procedure.
+    2. Choose a __Workspace__ to define the users and the available data sources that can interact with the procedure.
+    3. In the dropdown menu, choose a __Search Query__ you saved in Data Navigation.
+      The query defines the data you want to analyze with the analysis notebook. Verify that the data sources in the query are in the workspace you defined.
```
[the remark this answers is not in the document]
Comment (GrKuehnH) [reply handbook/preview-features/preview-features.md:25]: fixed.

File: handbook/preview-features/preview-features.md (hunk 1 above)
Comment (LukasKall) [reply handbook/preview-features/preview-features.md:11]: @GrKuehnH Do you also want to add the JupyterHub Module to the list of prerequisites?

**Review by StefanRR — APPROVED** [review]

File: handbook/preview-features/preview-features.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    4. Select the __Execution Mode__.
+      Select __Compare__ if you want to run the analysis notebook on all data elements the search query retrieves.
+      Select __Parallel__ if you want to run the analysis notebook on each data element the search query retrieves.
+4. Click __OK__.
+    The procedure uploads to Analysis Automation.
+5. Click the __Analysis Automation__ link to view the procedure on the __Procedures__ tab in Analysis Automation.
+6. Approve the procedure for it to go live.  
+*Note: Approving procedures requires a separate privilege.*
```
[the remark this answers is not in the document]
Comment (GrKuehnH) [reply handbook/preview-features/preview-features.md]: added priv name

File: handbook/preview-features/preview-features.md (hunk 3 above)
Comment (GrKuehnH) [reply handbook/preview-features/preview-features.md:16]: added screenshot

**Review by LukasKall — APPROVED** [review]

**Review by maxgax — APPROVED** [review]

**Review by jjaco2 — APPROVED** [review]
Reviewed this content earlier this year, outside of GitHub. Unless some significant technical changes have happened since then, it looks good to go.

**Pull request merged**